Gemfile

```diff
@@ -48,7 +51,7 @@ group :backend, :frontend, :core, :api do
   gem 'factory_bot_rails', '>= 4.8', require: false
 end
 
-group :backend, :frontend do
+group :frontend do
```

Comment:
Probably we can remove wrapping with `group :frontend` altogether. I mean, now, that's THE group :slightly_smiling_face:

Reply:
I removed the groups in the new update.